homeassistant/components/cover/isy994.py

```diff
@@ -0,0 +1,115 @@
+"""
+Support for ISY994 covers.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/cover.isy994/
+"""
+import logging
+from typing import Callable  # noqa
+
+from homeassistant.components.cover import CoverDevice, DOMAIN
+from homeassistant.components.isy994 import (ISYDevice, NODES, PROGRAMS, ISY,
```

Comment:
But ISY is not being imported - ISY is the instantiated client that is built off of `PyISY.ISY`

```python
global ISY

ISY = PyISY.ISY(...) # Create a new ISY client and assign it to the global ISY variable
```